src/itrim.cpp

```diff
@@ -122,17 +121,17 @@ usage() {
         "Options are:",
         " -M  admin - admin user uses this option to trim other users files",
         " -n  replNum - the replica number of the replica to be deleted",
-        " -N  numCopies - minimum number of most current copies to keep. The default",
+        " -N  numCopies - minimum number of good replicas to keep. The default",
         "     value is 2.",
         " -r  recursive - trim the whole subtree",
-        " -S  srcResource - specifies the resource of the replica to be deleted.",
-        "     If specified, only copies stored in this resource will be candidate",
+        " -S  srcResource - specifies the resource of the replica(s) to be deleted.",
+        "     If specified, only replicas stored in this resource will be candidates",
         "     for the deletion.",
         " -v  verbose",
         " -V  Very verbose",
-        "--age age_in_minutes - The minimum age of the copy in minutes for trimming.",
-        "     i.e., a copy will not be trimmed if its age is less.",
-        "--dryrun - Do a dry run. No copy will actually be trimmed.",
+        "--age age_in_minutes - The minimum age of the replica in minutes for trimming.",
+        "     i.e., a repliaa will not be trimmed if its age is less.",
```

Comment:
Dang. We missed one.

repliaa -> replica

Reply:
Same thing in the PR that was just merged.

Reply:
noooo.    on it.